pkg/backends/jmeter/jmeter.go

```diff
@@ -53,223 +45,54 @@ func New(
 	logger *zap.Logger,
 	namespacesLister coreListersV1.NamespaceLister,
 	reportURL string,
-	podAnnotations, namespaceAnnotations map[string]string,
+	podAnnotations, _ map[string]string,
```

Comment:
Why is `namespaceAnnotations` no longer used?

Reply:
Namespace are created by controller itself, so backends don't need it.
This will be removed on the future